scripts/change_url

```diff
@@ -17,7 +17,7 @@ old_domain=$YNH_APP_OLD_DOMAIN
 old_path=$YNH_APP_OLD_PATH
 
 new_domain=$YNH_APP_NEW_DOMAIN
-new_path=$YNH_APP_NEW_PATH
+new_path="/"
```

Comment:
Uuuuuh not sure to understand this change, can't the app be installed on a subpath ?

Reply:
oops, yes it does...